R/aes.R

```diff
@@ -430,7 +430,7 @@ alternative_aes_extract_usage <- function(x) {
 }
 
 extract_target_is_likely_data <- function(x, data, env) {
-  if (!is.name(x[[2]])) {
+  if (!is.name(x[[2]]) || identical(x[[2]], quote(.data))) {
```

Comment:
Couldn't find anything in rlang like `is_pronoun()` so we only escape `.data` here and not `.env`. I don't think `.env` is very common.